packages/kbn-test/src/functional_test_runner/lib/config/read_config_file.ts

```diff
@@ -52,8 +51,12 @@ async function getSettingsFromFile(log: ToolingLog, path: string, settingOverrid
     await cache.get(configProvider)!
   );
 
-  const logDeprecation = (error: string | Error) => log.error(error);
-  return transformDeprecations(settingsWithDefaults, logDeprecation);
+  if (get(settingsWithDefaults, 'servers.webdriver') !== undefined) {
```

Comment:
@spalger I didn't manage to find a config using `servers.webdriver`. Shouldn't we remove this logic?

Reply:
Yeah, we can just remove this. No need for deprecations in the FTR config

Reply:
removed